### ISSUE TITLE
TypeError: Object.fromEntries is not a function

### ISSUE THREAD

**Issue by JungleM0nkey** [synclounge/synclounge#335]
**Plex Media Server details**

- Version: 1.20.3.3483
- Operating System (Windows/Linux/Mac):  Ubuntu
- Running in Docker? (Yes/No): No
- Location (Local/Remote): Local
- Remote Access Enabled (Yes/No)?: Yes

**System Details**

- Operating System: Ubuntu Server 20.04.1

**SyncLounge Details**

- Version: 4.1.2
- Running in Docker? (Yes/No): No
- Using a Reverse Proxy (Yes/No): No
    If Yes, which Reverse Proxy (Nginx, Apache, Caddy, etc.):

**To Reproduce**

- Install synclounge with npm install -g synclounge
- Run synclounge
- Go to http://app.synclounge.tv
- Try to connect to custom URL server

**Describe your bug/issue**
Getting the following error on the synclounge server when a client tries to connect:
```
/usr/local/lib/node_modules/synclounge/node_modules/syncloungeserver/dist/socketserver/state.js:176
}) => Object.fromEntries([...rooms.get(roomId).users].filter(([socketId]) => socketId !== exceptSocketId).map(([id, data]) => [id, formatUserData(_objectSpread({
             ^

TypeError: Object.fromEntries is not a function
    at getOtherUserData (/usr/local/lib/node_modules/synclounge/node_modules/syncloungeserver/dist/socketserver/state.js:176:14)
    at getJoinData (/usr/local/lib/node_modules/synclounge/node_modules/syncloungeserver/dist/socketserver/state.js:203:12)
    at join (/usr/local/lib/node_modules/synclounge/node_modules/syncloungeserver/dist/socketserver/handlers.js:104:31)
    at Socket.socket.on.data (/usr/local/lib/node_modules/synclounge/node_modules/syncloungeserver/dist/socketserver/handlers.js:468:9)
    at Socket.emit (events.js:198:13)
    at /usr/local/lib/node_modules/synclounge/node_modules/socket.io/lib/socket.js:528:12
    at process._tickCallback (internal/process/next_tick.js:61:11)
```

**Comment by JungleM0nkey** [comment]
Got Plex updated to version 1.20.3.3483 from 1.19.5.3112 with no changes

**Comment by JungleM0nkey** [comment]
Fixed this by updating my nodejs from 10 to 14.15.0

Object.fromEntries() is not supported by nodejs below version 12

**Comment by nkrot** [comment]
@JungleM0nkey , could u please elaborate on how you updated nodejs as per your comment?
